### ISSUE TITLE
Allow setting CPU Model per guest and template

### ISSUE THREAD

**Issue by ET-Bent** [kimchi-project/kimchi#1092]
In many cases, it is needed to set the [CPU model](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Virtualization_Tuning_and_Optimization_Guide/sect-Virtualization_Tuning_Optimization_Guide-Virt_Manager-CPU_Options.html#sect-option_available_CPUs) of a guest to be able to use special CPU features in a this VM (e.g. nested virtualization). To change the CPU model you currently have to use `virt-manager` or `virsh`. This could be done within the panel.

Some ideas:

![auswahl_999 069](https://cloud.githubusercontent.com/assets/5788982/21298844/f808b7e0-c595-11e6-96d7-3e0d8accfd5f.png)

![auswahl_999 068](https://cloud.githubusercontent.com/assets/5788982/21298846/f9d97230-c595-11e6-9f99-660a86063080.png)

**Comment by alinefm** [comment]
Hi @ET-Bent 

Yeap! It would be very useful.

But I have some doubts about how to do that?
Do you want a way to set the CPU model freely, probably by a combox box with restricted options? Or do you want an option to "copy the host CPU configuration to guest" like virt-manager? Or both? =)

**Comment by ET-Bent** [comment]
@alinefm yes a combo-box would be the best option, providing options as seens in `virt-manager`. 

Please also include:

`host`
**and**
`host-passthrough`

Most panels leave out the `host-passthrough` option which is critically important when doing PCI passthrough.

Thanks for coming back to this issue!

**Comment by ET-Bent** [comment]
@alinefm Any updates on this?
